persistence/src/commonMain/db_user/migrations/68.sqm

```diff
@@ -0,0 +1,11 @@
+CREATE TRIGGER inserNewSelfClient
+AFTER INSERT ON Client
+WHEN new.user_id = (SELECT SelfUser.id FROM SelfUser LIMIT 1)
+BEGIN
+    INSERT INTO NewClient(id, device_type, registration_date, model)
+    VALUES(new.id, new.device_type, new.registration_date, new.model)
+    ON CONFLICT (id) DO UPDATE SET
+    device_type = coalesce(excluded.device_type, device_type),
+    registration_date = coalesce(excluded.registration_date, registration_date),
+    model = coalesce(excluded.model, model);
+END;
```

Comment:
one last thing since this fix should target RC having the migration file at 68 will not work we need to add it after the latest migration in RC